platform/javascript/export/export_plugin.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "export_plugin.h"
+#include "core/os/time.h"
```

Comment:
This should be:
```suggestion
#include "export_plugin.h"

#include "core/os/time.h"
```
As per https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes